rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1162,7 +1166,7 @@ def _load_model(
         model_dir: Text,
         finetune_mode: bool = False,
     ) -> "RasaModel":
-        file_name = meta.get("file")
+        file_name = meta["file"]
```

Comment:
should we raise a more specific error in case `file` is not in `meta`?

Reply:
Something like Rasa Error - Corrupt Metadata or so?

Reply:
What do you think of the new proposal?

Reply:
super weird that github doesn't seem to update this preview in this case... Never seen that before
here's the current code: 
https://github.com/RasaHQ/rasa/blob/97cae315c75d27c4a813bc8cf7719c26d09a39f8/rasa/nlu/classifiers/diet_classifier.py#L1163-L1178

Reply:
haha, works for me 👍🏻 Looks good!